languages/js/Makefile

```diff
@@ -1,46 +1,61 @@
-.PHONY: wasm install clean dev build fmtcheck typecheck lint parity test
+.PHONY: build test parity lint fmt fmtcheck typecheck docs repl clean install wasm
 
+# Build the WASM lib if it hasn't already been built; then build the TS lib.
 build: wasm
 	yarn build
 
+# Run the TS lib's test suite.
+test: install
```

Comment:
@Anniepoo I removed the new `jsbuildtest` Make target that we added, but `make test` should behave much better now. We don't actually need to build prior to running the tests (the `ts-jest` library handles that for us), and I implemented some smarter caching in the Makefile so that it isn't always building the WASM lib willy-nilly.